client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -68,6 +76,43 @@ class DebugTab extends Component {
 							</FormToggle>
 						</FormFieldset>
 						<div className="wp-super-cache__debug-fieldsets">
+							{ wp_cache_debug_log &&
+								<table>
```

Comment:
Ha. Yeah, this is actually a pet peeve of mine. 😛 This is probably more like a form than a table, albeit a non-editable one. It even has the submit button.